src/renderer/components/+preferences/preferences.tsx

```diff
@@ -45,9 +55,46 @@ export class Preferences extends React.Component {
     ]);
   }
 
+  onTabChange = (tabId: Pages) => {
+    this.activeTab = tabId;
+  };
+
+  renderNavigation() {
+    const extensions = appPreferenceRegistry.getItems().filter(e => !e.showInPreferencesTab);
+
+    return (
+      <Tabs className="flex column" scrollable={false} onChange={this.onTabChange} value={this.activeTab}>
+        <div className="header">Preferences</div>
+        <Tab value={Pages.Application} label="Application" data-testid="application-tab"/>
+        <Tab value={Pages.Proxy} label="Proxy" data-testid="proxy-tab"/>
+        <Tab value={Pages.Kubernetes} label="Kubernetes" data-testid="kube-tab"/>
+        <Tab value={Pages.Telemetry} label="Telemetry" data-testid="telemetry-tab"/>
```

Comment:
Probably better to have a list of `{pageId: string, label: string, testId?: string, content: React.ReactNode}` and render them via `[].map()`?

Reply:
Yes, will change it on next iteration 👍🏻